.github/workflows/app-build-verify.yml

```diff
@@ -63,181 +63,191 @@ jobs:
-
-        cd tests/apps/verify-${{ inputs.framework }}
-        briefcase create windows VisualStudio
+        briefcase create windows VisualStudio \
+          ${{ steps.output-format.outputs.template-override }}
         briefcase build windows VisualStudio
         briefcase package windows VisualStudio --adhoc-sign
 
-    - name: Build Linux System project (Ubuntu, local)
+    - name: Build Linux System Project (Ubuntu, local)
       if: >
         startsWith(inputs.runner-os, 'ubuntu')
-        && startsWith(inputs.python-version, steps.system-python.outputs.python-version)
+        && startsWith(inputs.python-version, steps.system-python.outputs.version)
         && contains(fromJSON('["", "Linux"]'), inputs.target-platform)
         && contains(fromJSON('["", "system"]'), inputs.target-format)
+      working-directory: ${{ steps.create.outputs.project-path }}
       run: |
-        sudo apt-get update -y && sudo apt-get install -y python3-dev python3-pip libcairo2-dev libgirepository1.0-dev
-        cd tests/apps/verify-${{ inputs.framework }}
-        briefcase create linux system
+        sudo apt update -y
+        sudo apt install -y --no-install-recommends python3-dev python3-pip libcairo2-dev libgirepository1.0-dev
+
+        briefcase create linux system \
+          ${{ steps.output-format.outputs.template-override }}
         briefcase build linux system
         briefcase package linux system --adhoc-sign
 
-    - name: Build Linux System project (Debian, Dockerized)
+    - name: Build Linux System Project (Debian, Dockerized)
       if: >
         startsWith(inputs.runner-os, 'ubuntu')
         && contains(fromJSON('["", "Linux"]'), inputs.target-platform)
         && contains(fromJSON('["", "system"]'), inputs.target-format)
+      working-directory: ${{ steps.create.outputs.project-path }}
       run: |
-        cd tests/apps/verify-${{ inputs.framework }}
-        briefcase create linux system --target debian:bullseye
+        briefcase create linux system --target debian:bullseye \
+          ${{ steps.output-format.outputs.template-override }}
         briefcase build linux system --target debian:bullseye
         briefcase package linux system --target debian:bullseye --adhoc-sign
 
-    - name: Build Linux System project (RPM, Dockerized)
-      # fedora:37 ships with Python 3.11 and PySide2 cannot be installed
+    - name: Build Linux System Project (RPM, Dockerized)
       if: >
         startsWith(inputs.runner-os, 'ubuntu')
-        && !contains(fromJSON('["PySide2"]'), inputs.framework)
         && contains(fromJSON('["", "Linux"]'), inputs.target-platform)
         && contains(fromJSON('["", "system"]'), inputs.target-format)
+      working-directory: ${{ steps.create.outputs.project-path }}
       run: |
-        cd tests/apps/verify-${{ inputs.framework }}
-        briefcase create linux system --target fedora:37
+        briefcase create linux system --target fedora:37 \
```

Comment:
Doesn't need to be fixed in this PR, but flagging that Fedora 37 is EOL in 2 weeks, so we're likely going to need to bump this to 38 or 39 soon.

Reply:
👍🏼 I should take care of this in my next round of PRs. I think some of the versions in the templates (e.g. for `org.kde.Platform`) need bumps as well.